update!: Windows guide (#1659)
privacyguides/privacyguides.org: docs/windows/overview.md

### Installing Windows

We recommend that you use the official [Media Creation tool](https://www.microsoft.com/software-download/windows11) to flash the ISO to the USB, over third-party options such as Rufus, Balena Etcher, etc., so that you don't tamper the ISO.

#### Downloading ISO

To download the ISO. Follow these steps :

- Download Media Creation tool under `Windows 11 Installation Media`
- Open a Command prompt terminal in the directory where `mediacreationtool.exe` is downloaded.
- And Input the following Command :
    ```
    mediacreationtool.exe /Eula Accept /Retail /MediaArch x64 /MediaLangCode en-US /MediaEdition Enterprise
    ```
- If it asks for Activation key, Use this Generic Key `XGVPP-NMH47-7TTHJ-W3FW7-8HV2C`. This will just allow you to download the ISO but activation is totally upon the user.
- Accept the UAC prompt
- Download the ISO file or flash to a USB as you wish

!!! info "Note"

>     - The ISO will consists **only** of Professional, Education & Enterprise edition with a size of ~4.2 GB (Instead of >5.5GB when you download the Multi-Edition ISO) when you download using the above way no other editions such as Home included in it.

Reviewer: ```suggestion
    - The ISO will consist **only** of Professional, Education & Enterprise editions with a size of ~4.2 GB (Instead of >5.5GB when you download the Multi-Edition ISO). When you download using the above way, no other editions such as Home are included in it.
```

>     - If you want to change the Language of the ISO file, Just change the `en-US` part with the appropriate language and country code as per your needs.

Reviewer: ```suggestion
    - If you want to change the language of the ISO file, Just change the `en-US` part with the appropriate language and country code as per your needs.
```

### Activating Windows

Activating Education/Enterprise edition is different because for Enterprise Edition it needs to be a part of an enterprise network or buying an enterprise License for several devices and use it for your one device & for Education Edition it needs to be a part of school network or managed by a school administrator.

For activating Professional edition, you can buy the license key from resellers (not recommended) or the [Microsoft Store](https://www.microsoft.com/d/windows-11-pro/dg7gmgf0d8h4?rtc=1).

If you are currently using Pro and want to upgrade to Enterprise. Then, Follow the guide [here](https://www.kapilarya.com/how-to-upgrade-windows-11-pro-to-enterprise-edition)

!!! abstract "Note"
    This guide will be mostly on Windows 11 but some of the recommendations can be applied to Windows 10 too.

!!! danger "Warning"

>     If you are going to install Windows 11, Then install it only on supported devices and it is not recommended to use tools/scripts that are available online to bypass the requirements which totally breaks the security of Windows 11 which it is aimed for.

Reviewer: ```suggestion
    If you are going to install Windows 11, then install it only on supported devices. It is not recommended to use tools/scripts that are available online to bypass the requirements which would break the security of Windows 11.
```

>     Never download **Pirated ISO Files**

Reviewer: I feel like the emphasis here should be on **Never**
